Source/Session/Session.swift

```diff
@@ -10,7 +10,9 @@ public class Session: NSObject {
     
     public let webView: WKWebView
     public var pathConfiguration: PathConfiguration?
-    
+
+    var needsCacheClearing = false
```

Comment:
Could this be made public or could we expose a method on `Session` to mark is as needing clearing?

We're building on top of Turbo iOS in the React Native adapter and it'd be great to be able to mark session snapshots as stale.